tools/pylint-extensions/azure-pylint-guidelines-checker/setup.py

```diff
@@ -6,7 +6,7 @@
 
 setup(
     name="azure-pylint-guidelines-checker",
-    version="0.0.8",
+    version="0.1.0",
```

Comment:
Is this the version we want for a bug fix?

Reply:
I was going to start aligning with how python version bumps work with Breaking.Major.Minor, I thought a bug fix could be major, but can make it minor? So I can make it .0.0.9 then

Reply:
semvar uses MAJOR.MINOR.PATCH. This feels like a patch to me.